src/java/org/apache/cassandra/db/virtual/AbstractMutableVirtualTable.java

```diff
@@ -57,7 +57,7 @@ protected AbstractMutableVirtualTable(TableMetadata metadata)
     }
 
     @Override
-    public final void apply(PartitionUpdate update)
+    public void apply(PartitionUpdate update)
```

Comment:
this might go back if we prefer thresholds to be on a vtable with a column of tuple type.